forest/src/daemon.rs

```diff
@@ -60,6 +93,16 @@ pub(super) async fn start(config: Config) {
     let mut db = RocksDb::new(config.data_dir + "/db");
     db.open().unwrap();
     let db = Arc::new(db);
+
+    // Sync from snapshot
```

Comment:
If I understand correctly we dont actually sync from snapshot rather we set the snapshot? Reason I mention this is because I dont think any sync logic is called in import_chain?

Reply:
Well, we populate our blockstore from the snapshot file. And furthermore, we can also recompute all the state if we import with `--import-chain`.

Reply:
Yeah if you're wondering why not full block valation, that is intended, it's just calculating intermediate states and validating those roots are correct (full block validation would be too slow when you want to trust the export)